bin/resources/GameIndex.yaml

```
@@ -2130,51 +2140,82 @@ SCCS-40005:
     halfPixelOffset: 1 # Fixes effect misalignment.
     moveHandler: "MV_Ico" # Fixes depth buffer post-processing.
 SCCS-40006:
-  name: "Zhen Sanguo Wushuang 2"
+  name: 真·三国無双2
+  name-sort: "Zhen Sanguo Wushuang 2"
+  name-en: "Zhen Sanguo Wushuang 2"
   region: "NTSC-C"
 SCCS-40007:
-  name: "Arc the Lad - Seirei no Tasogare"
+  name: 亚克传承　～精灵之黄昏～
+  name-sort: "Yake Chuancheng Jingling Zhi Huanghun"
+  name-en: "Arc the Lad - Seirei no Tasogare"
   region: "NTSC-C"
 SCCS-40009:
-  name: "Dragon Ball Z 2"
+  name: 龙珠二世
```

```suggestion
  name: "龙珠二世"
```

Same for other serials, should be fine then.